documentation/book/proc-deploying-kafka-cluster-kubernetes.adoc

```diff
@@ -5,26 +5,28 @@
 [id='deploying-kafka-cluster-kubernetes-{context}']
 = Deploying the Kafka cluster to {KubernetesName}
 
+The following procedure describes how to deploy an ephemeral or persistent Kafka cluster to {KubernetesName} on the command line.
+
 .Prerequisites
 
-* Before deploying a Kafka cluster, the Cluster Operator must be deployed.
+* You must have deployed the Cluster Operator before you can deploy a cluster.
```

Comment:
I wonder if this is the right form. Its in _Prerequisites_, so it should be clear that this should be done before. We might not write it again. Also, I think the style differs in all  other procedures. Why not just:

_* Cluster Operator is deployed._